src/components/molecules/VenueChat/VenueChat.tsx

```diff
@@ -24,15 +23,13 @@ interface ChatOutDataType {
 }
 
 const VenueChat: FC = () => {
-  useConnectVenueUsers();
-
```

Comment:
I have moved this kind of listeners to hooks. IMO A good approach is to have a listener only there, where you select the data you have subscribed to. So, basically, one should only use them in conjunction with `useSelect`. I have unified this kind of logic and moved it to hooks. It makes it easier to interact with data as you will never forget to subscribe to data.

It will neither do bad or good. It is just useless.

Reply:
That makes sense (see also https://github.com/sparkletown/sparkle/pull/1042#discussion_r551814049 for similar explanation)

If you are happy that you have found and updated every bit of code that was relying on this old 'external connect hook' pattern, and that nothing will inadvertently break because of removing this, then happy for you to mark this as resolved.